DDR4 OC Guide.md

```diff
@@ -479,16 +501,18 @@ The default value is fixed 1.100V and AMD recommends keeping it at that level. I
    * On Intel, tWTRS/L should be left on auto and controlled with tWRRD_dg/sg respectively. Dropping tWRRD_dg by 1 will drop tWTRS by 1. Likewise with tWRRD_sg. Once they're as low as you can go, manually set tWTRS/L.
    * On Intel, changing tCWL will affect tWRRD_dg/sg and thus tWTR_S/L. If you lower tCWL by 1 you need to lower tWRRD_dg/sg by 1 to keep the same tWTR values. Note that this might also affect tWR per the relationship described earlier.
    * <sup>1</sup>Some motherboards don't play nice with odd tCWL. For example, I'm stable at 4000 15-19-19 tCWL 14, yet tCWL 15 doesn't even POST. Another user has had similar experiences. Some motherboards may seem fine but have issues with it at higher frequencies (Asus). Manually setting tCWL equal to tCL if tCL is even or one below if tCL is uneven should alleviate this (eg. if tCL = 18 try tCWL = 18 or 16, if tCL = 17 try tCWL = 16).
-   * The extreme preset is not the minimum floor in this case. tRTP can go as low as 6, while tWTRS/L can go as low as 1 6. Some boards are fine doing tCWL as low as tCL - 6. Keep in mind that this *will* increase the load on your memory controller.
-   * On AMD, tCWL can often be set to tCL - 2 but is known to require higher tWRRD.
+   * The extreme preset is not the minimum floor in this case. tRTP can go as low as 5 (6 with Gear Down Mode on), while tWTRS/L can go as low as 1/6. Some boards are fine doing tCWL as low as tCL - 6. Keep in mind that this *will* increase the load on your memory controller.
+   * On AMD, tCWL can often be set to tCL - 2 but is known to require higher tWRRD. Read to Write burst gap is tCWL-tCL + tRDWR. Since this value normally has a set threshold fr stability, increase the gap between tCWL and tCL will require a higher tRDWR to compensate.
    
 4. Now for the tertiaries:
     * If you're on AMD, refer to [this post](https://redd.it/ahs5a2).  
       My suggestion:
   
        | Timing | Safe | Tight | Extreme |
        | ------ | ---- | ----- | ------- |
-       | tRDRDSCL tWRWRSCL | 4 4 | 3 3 | 2 2 |
+       | tRDRDSCL tWRWRSCL | 5 5 | 4 4 | 3 3 |
```

Comment:
Don't think I've seen an IC that couldn't do SCLs at 4 4 apart from Micron Rev E at 5000

Reply:
I've seen it with CJR and Rev E. 5 vs 4 doesnt really even bring that big a gain tbh, but sure

Reply:
I'll change it back